Snowflake.Data/Core/Session/SessionPool.cs

```diff
@@ -41,13 +42,14 @@ private SessionPool()
             _poolConfig = new ConnectionPoolConfig();
         }
 
-        private SessionPool(string connectionString, SecureString password, ConnectionPoolConfig poolConfig)
+        private SessionPool(string connectionString, SecureString password, ConnectionPoolConfig poolConfig, string connectionStringWithoutSecrets)
```

Comment:
I'd rather had this removal of secrets inside and not have it as partially redundant parameter, is this possible?

Reply:
we agreed to leave as is